commit 87a7adb1ad587e4b59c15c3caaec8d770b9668ae (refs/changes/59/559/3)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-10-03T18:40:19+00:00 (3 years ago)
    
    CNS-161 retry backoff for changefeed should use a finite timeout
    CNS-162 updater cueball agent should use higher maximum socket cap
    Reviewed by: Richard Kiene <richard.kiene@joyent.com>
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/updater.js b/updater.js
index fd615b7..6ff0b6a 100644
--- a/updater.js
+++ b/updater.js
@@ -46,7 +46,7 @@ var log = bunyan.createLogger({name: 'cns',
 var agent = new cueball.HttpAgent({
 	resolvers: [conf.binder_domain],
 	spares: 2,
-	maximum: 4,
+	maximum: 10,
 	recovery: {
 		default: {
 			timeout: 2000,
@@ -97,8 +97,8 @@ var cfOpts = {
 		subResources: conf.changefeed_opts.subResources
 	},
 	backoff: {
-		maxTimeout: Infinity,
-		minTimeout: 10,
+		maxTimeout: 10000,
+		minTimeout: 2000,
 		retries: Infinity
 	}
 };
